File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/exponential/ExpZ.java
package net.fwitz.math.main.binary.complex.functions.exponential;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class ExpZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("exp(z)")
                .computeFn(Complex::exp)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/exponential/ExpZ.kt
package net.fwitz.math.main.binary.complex.functions.exponential

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object ExpZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("exp(z)")
        .computeFn(Complex::exp)
        .render()
}